backend/inspirehep/indexer/base.py

```diff
@@ -132,3 +134,62 @@ def bulk_action(self, record_uuid):
             LOGGER.exception(
                 "Kombu is not able to process response!", uuid=str(record_uuid)
             )
+
+
+class InspireRecordFulltextIndexer(InspireRecordIndexer):
```

Comment:
isn't it `LiteratureRecordFulltextIndexer`?

Reply:
yes, would be better probably